packages/standard-generator/src/fs.config.js

```diff
@@ -36,7 +36,7 @@ module.exports = {
     },
     {
       modulePath: '@jpmorganchase/mosaic-plugins/SearchIndexPlugin',
-      options: { maxLineLength: 240, maxLineCount: 240, keys: ['description'] }
+      options: { maxLineLength: 240, maxLineCount: 240 }
```

Comment:
does this need changing in site as well ?

Reply:
Nope, site will pick it up from here 👍

Reply:
ah right